src/App.tsx

```diff
@@ -1,19 +1,80 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useCallback,
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+
+import { getTodos } from './api/todos';
+
 import { AuthContext } from './components/Auth/AuthContext';
+import { ErrorMessages } from './components/ErrorMessages';
+import { TodosFilter } from './components/TodosFilter/TodosFilter';
+import { NewTodo } from './components/NewTodo/NewTodo';
+import { TodosList } from './components/TodosList/TodosList';
+
+import { FilterStatus } from './types/FilterStatus';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [visibleTodos, setVisibleTodos] = useState<Todo[]>(todos);
+  const [filterBy, setFilterBy] = useState<FilterStatus>(FilterStatus.ALL);
+  const [hasError, setHasError] = useState(false);
+
   const user = useContext(AuthContext);
   const newTodoField = useRef<HTMLInputElement>(null);
 
+  const loadUserTodos = useCallback(async () => {
+    if (user) {
+      try {
+        const todosFromServer = await getTodos(user.id);
+
+        setTodos(todosFromServer);
+      } catch {
+        setHasError(true);
+      }
```

Comment:
You should add some message that describes the error that can occur when we load todos